web/src/views/ConsentPortal/OpenIDConnect/DecisionFormClaims.tsx

```
@@ -0,0 +1,103 @@
+    );
+
+    const hasClaims = props?.essential_claims || props?.claims;
+
+    return (
+        <Fragment>
+            {hasClaims ? (
+                <Grid size={{ xs: 12 }}>
+                    <Box className={classes.container}>
+                        <List className={classes.list}>
+                            {props.essential_claims?.map((claim: string) => (
+                                <Tooltip title={translate("Claim", { name: claim })}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)